specifyweb/frontend/js_src/lib/components/Atoms/className.ts

```diff
@@ -74,13 +75,13 @@ export const className = {
   containerFullGray: `${containerFull} bg-[color:var(--form-background)]`,
   containerBaseUnstyled,
   containerBase,
-  containerCenter: `${containerBase} max-w-[min(100%,var(--form-max-width))]
+  containerCenter: `${containerBase} max-w-[var(--form-max-width)]
```

Comment:
w3 was complaining about using `min()` in `max-width`

setting max-width to 100% does not make any impact (hopefully), thus it's redundant and can be simplified

(in my limited testing in chrome, this had no impact at different breakpoints with different `--form-max-width` values)

though, this could have been the cause of the infamous form shrinking bug that @grantfitzsimmons was encountering in the past (https://github.com/specify/specify7/issues/1954)